ncs/Kconfig

```diff
@@ -29,3 +29,27 @@ config SUIT_ENVELOPE_OUTPUT_MPI_MERGE
 config SUIT_LOCAL_ENVELOPE_GENERATE
   bool "Generate local envelope"
   default y if SOC_NRF54H20_CPUAPP || SOC_NRF54H20_CPURAD
+
+config SUIT_EXTRACT_IMAGE_TO_DFU_CACHE
+  bool "Extract firmware image to DFU cache"
```

Comment:
Kconfig indent is a tab, help text is 1x tab and 2x spaces

Reply:
It seems that the whole file used incorrect indents, so fixed everywhere in this file